pkg/cmd/system/prune.go

```diff
@@ -0,0 +1,78 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package system
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/nerdctl/pkg/api/types"
+	"github.com/containerd/nerdctl/pkg/cmd/builder"
+	"github.com/containerd/nerdctl/pkg/cmd/container"
+	"github.com/containerd/nerdctl/pkg/cmd/image"
+	"github.com/containerd/nerdctl/pkg/cmd/network"
+	"github.com/containerd/nerdctl/pkg/cmd/volume"
+)
+
+// Prune will remove all unused containers, networks, images (both dangling and unreferenced), and optionally, volumes.
```

Comment:
With https://github.com/containerd/nerdctl/pull/2036, `images (both dangling and unreferenced)` is no longer true (depends on the `All` value). You can update the comment in either one of the PRs.

Reply:
Yep.

Reply:
Updated in this PR.